CMakeLists.txt

```diff
@@ -19,13 +19,26 @@ endif ()
 
 target_include_directories(uwuEngine PUBLIC ${FMOD_INCLUDE_DIR})
 target_include_directories(uwuEngine PUBLIC ${GLEW_INCLUDE_DIR})
+target_include_directories(uwuEngine PUBLIC lib/**/include)
```

Comment:
Not sure if this line is neccessary or not. There is a line above that should be doing the same thing.